server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -555,12 +554,6 @@ public FlowInstanceDetailResp detail(@NotNull Long id) {
     @Transactional(rollbackFor = Exception.class)
     public FlowInstanceDetailResp cancel(@NotNull Long id, Boolean skipAuth) {
         FlowInstance flowInstance = mapFlowInstance(id, flowInst -> flowInst, skipAuth);
-        if (!skipAuth) {
```

Comment:
this auth process is already done by `mapFlowInstance`, It is not necessary to do it again